webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -557,49 +566,59 @@ private static void addFmcSolutionSheet(PdfWriter docWriter, Document doc, Scram
         if(showScrambleCount) {
             offsetTop -= fontSize + 2;
         }
+        
+        String tempMessage = ""; // saves typing in i18n
```

Comment:
I don't understand the purpose of this variable or the comment here, could you explain what it's about?

Reply:
FitText requires the whole string to compute size, then we have to render that very same string. Instead of typing it [again](https://github.com/campos20/tnoodle/blob/master/webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java#L576) and [again](https://github.com/campos20/tnoodle/blob/master/webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java#L577), tempMessage holds the string and can be used twice without verbose typing.

Reply:
Ahh, I see now! Could we create a method to handle this for us? Maybe a `fitAndShowText()` method?

Reply:
Done.